test/functional/passwords_controller_test.rb

```diff
@@ -67,6 +68,7 @@ class PasswordsControllerTest < ActionController::TestCase
     setup do
       @user = create(:user)
       @user.forgot_password!
+      @controller.session[:mfa_expires_at] = 15.minutes.from_now.to_s
```

Comment:
I like in the webauthn case that we're using the `edit` action to set the expiry https://github.com/rubygems/rubygems.org/blob/b3137d279208708dca56a01ad17a71d41895fcf3/test/functional/passwords_controller_test.rb#L131.

Can we do the same here?